src/components/AssetScanner/stories/full.md

```diff
@@ -4,23 +4,23 @@
 
 #### Description:
 
-Using for recognising assets name strings from photo.
+Uses for recognising assets name strings from captured image.
 
 #### Usage requirements:
 
-Component depends on few third-part APIs, which is provided by Google
-Vision (to recognize text on picture). The user has to insert props
-to the component with a valid API key for Google Vision. Follow this
-guide to generate an api-key:
+By default, component uses Google Vision API (to recognize text on picture). User has to provide valid API key for Google Vision via **ocrKey** prop to use embedded OCR functional.
+Please, follow this guide to generate an api-key:
 
 **Google Vision API**
 
-> Before you can use the Cloud Vision API, you must enable it for your project and generate your API key, see here how – [Enable the Vision API](https://cloud.google.com/vision/docs/before-you-begin).   
+> Before you can use the Cloud Vision API, you must enable it for your project and generate your API key, see here how – [Enable the Vision API](https://cloud.google.com/vision/docs/before-you-begin).
```

Comment:
```suggestion
> Before you can use the Cloud Vision API, you must enable it in your project and generate your API key, see here how – [Enable the Vision API](https://cloud.google.com/vision/docs/before-you-begin).
```

Reply:
"for" means that you have to enable Google Vision option in your Google Cloud account exactly FOR your project.

Reply:
got it